steps/27/README.md

```diff
@@ -43,7 +43,7 @@ Finally, we perform our assertions. We check each branch of the formatter logic
 ```ts
 import ResourceModel from "sap/ui/model/resource/ResourceModel";
 import Controller from "sap/ui/core/mvc/Controller";
-import formatter from "../../../model/formatter";
+import formatter from "ui5/walkthrough/model/formatter";
```

Comment:
Why do I need to add the "namespace" to the formatter path? 
I think this should be explained here as well so learners can extrapolate from here to their use case.

Is this related to the fact that a new namespace is defined in Test.qunit.html?
If so it mightbe easier then to first create Test.qunit.html before creating unit/model/formatter.ts?